terraform/prow_clusters/prow_clusters.tf

```diff
@@ -0,0 +1,149 @@
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Multi repo standard clusters
+module "multi-repo-stable" {
+  source = "../testgroup"
+  channel = "stable"
+}
+
+module "multi-repo-regular" {
+  source = "../testgroup"
+  channel = "regular"
+}
+
+module "multi-repo-rapid" {
+  source = "../testgroup"
+  channel = "rapid"
+}
+
+module "multi-repo-rapid-latest" {
+  source = "../testgroup"
+  channel = "rapid"
+  min_master_version = "latest"
+}
+
+module "multi-repo-psp" {
+  source = "../testgroup"
+  channel = "regular"
+  suffix = "psp"
+}
+
+module "multi-repo-bitbucket" {
+  source = "../testgroup"
+  channel = "regular"
+  suffix = "bitbucket"
+}
+
+module "multi-repo-gitlab" {
+  source = "../testgroup"
+  channel = "regular"
+  suffix = "gitlab"
+}
+
+# Multi repo autopilot clusters
+module "multi-repo-autopilot-stable" {
+  source = "../testgroup_autopilot"
+  channel = "stable"
+}
+
+module "multi-repo-autopilot-regular" {
+  source = "../testgroup_autopilot"
+  channel = "regular"
+}
+
+module "multi-repo-autopilot-rapid" {
+  source = "../testgroup_autopilot"
+  channel = "rapid"
+}
+
+module "multi-repo-autopilot-rapid-latest" {
+  source = "../testgroup_autopilot"
+  channel = "rapid"
+  min_master_version = "latest"
+}
+
+# Mono repo clusters
+module "mono-repo-stable" {
+  source = "../testgroup"
+  prefix = "mono-repo"
+  channel = "stable"
+  num_clusters = 3
+}
+
+module "mono-repo-regular" {
+  source = "../testgroup"
+  prefix = "mono-repo"
+  channel = "regular"
+  num_clusters = 3
+}
+
+module "mono-repo-rapid" {
+  source = "../testgroup"
+  prefix = "mono-repo"
+  channel = "rapid"
+  num_clusters = 3
+}
+
+module "mono-repo-rapid-latest" {
+  source = "../testgroup"
+  prefix = "mono-repo"
+  channel = "rapid"
+  min_master_version = "latest"
+  num_clusters = 3
+}
+
+# One off clusters
+module "multi-repo-kind" {
+  source = "../e2ecluster"
+  name = "multi-repo-kind"
+  channel = "regular"
+  subnetwork = google_compute_subnetwork.e2e-subnetwork-1.name
+  network = google_compute_network.e2e-network.name
+}
+
+module "mono-repo-kind" {
+  source = "../e2ecluster"
+  name = "mono-repo-kind"
+  channel = "regular"
+  subnetwork = google_compute_subnetwork.e2e-subnetwork-1.name
+  network = google_compute_network.e2e-network.name
+}
+
+module "multi-repo-kcc" {
+  source = "../e2ecluster"
+  name = "multi-repo-kcc"
+  channel = "regular"
+  subnetwork = google_compute_subnetwork.e2e-subnetwork-1.name
+  network = google_compute_network.e2e-network.name
+}
+
+module "multi-repo-gcenode" {
```

Comment:
For the gcenode cluster, the workload identity should be **disabled**.  We need to make `workload_identity_config` configurable.

Reply:
Done